.travis.yml

```diff
@@ -46,6 +46,19 @@ matrix:
         - DESC="findbugs and pmd"
         - CMD="mvn clean compile pmd:check findbugs:check"
 
+    # checkstyle injection compile no error (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle injection compile no error"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cp src/main/resources/com/github/checkstyle/regression/extract/ExtractInfoGeneratorTest.java "
+        - CMD3="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/ExtractInfoGeneratorTest.java &&"
+        - CMD4="cp src/main/resources/com/github/checkstyle/regression/extract/Json.java "
+        - CMD5="checkstyle/src/test/java/com/puppycrawl/tools/checkstyle/Json.java &&"
+        - CMD6="cd checkstyle &&"
+        - CMD7="mvn clean compile"
+        - CMD="$CMD1$CMD2$CMD3$CMD4$CMD5$CMD6$CMD7"
```

Comment:
I am fine with this being separated.
I don't expect this to be done in #50 in the junits (if that was what your thinking), as junits won't be cloning the external repository as part of their tests. They should be pretty quick tests, which is why missing code coverage for this one script is fine. That is why travis makes better sense for doing all these long complicated tasks.

Once you add this to travis I will merge this as it will be confirmed working.